### DISCUSSION TITLE
How to open file with Sublime-Text as an editor?

### DISCUSSION THREAD

**Discussion by vinoff** [Murmele/Gittyup#380] [Q&A]
I am trying to open files, for editing, with sublime-text. I added "subl" to "Options -> Tools -> External Editor". However, when I do, for example, "Edit working Copy", it won't open Sublime-Text. It still opens it in Gittyup's editor.. What am I doing wrong?

EDIT: If I do "Edit in External Editor", it works as expected. Why is "Edit Working Copy", "Edit New Revision" and "Edit Old Revision" not calling the external editor?

**Comment by exactly-one-kas** [comment]
There's not really a reason for that, it's just been that way
